### DISCUSSION TITLE
When selecting a file in a git repo to compare to an older version

### DISCUSSION THREAD

**Discussion by ScottStauffer** [tabulartools/pbi-explorer#52] [Q&A] [answered]
It seems to show the newer version on the left and the older version of the right...
this makes it look like I'm removing visualizations and m code instead of adding it...
perhaps it is my configuration.  Is there an easy way to chose a different commit hash for a left of right pane, or swap versions from side to side?

**Comment by ScottStauffer** [comment]
![image](https://github.com/tabulartools/pbi-explorer/assets/25377151/e4da6eaf-944d-49e2-b5f5-612fde360d8a)

**Reply by ScottStauffer** [reply]
commit [dba9d69] is newer than [3c3e3e4] and I'd expect it to be on the right and the stuff highlighted in red, to be instead highlighted in green.

BTW, I'm *_Super_* excited to see the git integration.  This adds to my PASS Data Community Summit Session.

**Comment by DBojsen** [comment, marked as answer]
Hi @ScottStauffer 

Thanks for reporting. 
We are aware - it’s on the to-do list. 
The plan is to use commit datetime and file save datetime to deduct which goes where

**Reply by ScottStauffer** [reply]
THANKS!

**Comment by DBojsen** [comment]
Hi @ScottStauffer 

Did you see that we released this in December?

BR David - Tabular Tools
